.changeset/two-kings-study.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Changeset seems empty.

Reply:
As I know, we don't put changeset info, if it's small addition